src/core/Form/Textarea/Textarea.baseStyles.tsx

```diff
@@ -1,86 +1,94 @@
 import { css } from 'styled-components';
-import { TextareaProps } from './Textarea';
-import { withSuomifiTheme, TokensAndTheme } from '../../theme';
+import { defaultThemeTokens as theme } from '../../theme';
 import { disabledCursor } from '../../../components/utils/css';
 import { element, font } from '../../theme/reset';
 import { absoluteFocus } from '../../theme/utils/focus';
-import { Omit } from '../../../utils/typescript';
```

Comment:
This remove is good as TypeScript has the **Omit** nowadays. 

Maybe creating ticket to backlog to remove the use the custom Omit also in other places so we can get rid of it altogether.

Reply:
There were only three cases left. Those are now removed with the definition.